src/templates/_common/includes/mfsa.jsx

```diff
@@ -7,7 +7,8 @@ const MFSA = () => {
     return (
         <div style={style} className='container mfsa_message'>
             <div className='notice-msg center-text gr-parent gr-child'>
-                <p>{it.L('In the EU, financial binary options are only available to professional investors.')}</p>
+                <p>{it.L('In the EU, financial binary options are only available to professional investors.')}&nbsp;<a id='professional-cta' className='invisible' href={it.url_for('user/settings/professional')}>{it.L('Apply now as a professional investor')}</a>
```

Comment:
better to have a CTA for `Apply now as a professional investor` , please get help from UI team